### PR TITLE
fix ownership for nf-test and adding est-sfs module (#4049)

### PR DESCRIPTION
**Pull request by BioInf2305**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/estsfs/main.nf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    conda "bioconda::est-sfs=2.04"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/est-sfs:2.04--h245ed52_0':
+        'biocontainers/est-sfs:2.04--h245ed52_0' }"
+
+    input:
+    tuple val(meta), path(e_config), path(data), path(seed)
+
+    output:
+    tuple val(meta), path("${prefix}_sfs.txt"), emit: sfs_out
```
Comment (FriederikeHanssen) [inline modules/nf-core/estsfs/main.nf:16]:
```suggestion
    tuple val(meta), path("${prefix}_sfs.txt")   , emit: sfs_out
```

File: modules/nf-core/estsfs/main.nf [hunk 2]
```diff
@@ -0,0 +1,49 @@
+process ESTSFS {
+    tag "$meta.id"
+    label 'process_low'
+
+
```
Comment (FriederikeHanssen) [inline modules/nf-core/estsfs/main.nf:5]:
```suggestion
```

File: tests/config/test_data.config [hunk 3]
```diff
@@ -634,6 +634,11 @@ params {
                 mock_sam                        = "${params.test_data_base}/data/genomics/homo_sapiens/pairtools/mock.sam"
                 frag_bed                        = "${params.test_data_base}/data/genomics/homo_sapiens/pairtools/frag.bed"
             }
+	   'estsfs'{
+		config_file                     = "${params.test_data_base}/data/delete_me/estsfs/config-JC.txt"
+		data_file                       = "${params.test_data_base}/data/delete_me/estsfs/TEST-DATA.TXT"
+		seed_file                       = "${params.test_data_base}/data/delete_me/estsfs/seedfile.txt"
```
Comment (FriederikeHanssen) [inline tests/config/test_data.config:640]:
```suggestion
		    config_file                     = "${params.test_data_base}/data/delete_me/estsfs/config-JC.txt"
		    data_file                       = "${params.test_data_base}/data/delete_me/estsfs/TEST-DATA.TXT"
		    seed_file                       = "${params.test_data_base}/data/delete_me/estsfs/seedfile.txt"
```

**Comment by drpatelh** [comment]
Bumping to @maxulysse for review please 🙏🏽

**Review by maxulysse — APPROVED** [review]

**Pull request merged**